MSAL/src/requests/MSALSilentRequest.m

```diff
@@ -187,7 +187,8 @@ - (BOOL)isServerUnavailable:(NSError *)error
         return NO;
     }
     
-    return ([error code] >= 500 && [error code] <= 599);
+    NSInteger errorCode = [[error.userInfo objectForKey:MSIDHTTPResponseCodeKey] intValue];
```

Comment:
please also check for actual error code to be unhandled response for completeness

Reply:
done